Title: 294161 – segfault in "${PORTAGE_BIN_PATH}"/filter-bash-environment.py (merging kde-base/plasma-workspace-4.3.3-r1)
URL: https://bugs.gentoo.org/show_bug.cgi?id=294161

Go to: 
		Gentoo Home
Documentation
Forums
Lists
Bugs
Planet
Store
Wiki
Get Gentoo!





Gentoo's Bugzilla – Bug 294161
segfault in "${PORTAGE_BIN_PATH}"/filter-bash-environment.py (merging kde-base/plasma-workspace-4.3.3-r1)
Last modified: 2009-11-23 12:30:28 UTC node [vulture]


Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]













Bug 294161 
      - segfault in "${PORTAGE_BIN_PATH}"/filter-bash-environment.py (merging kde-base/plasma-workspace-4.3.3-r1)


Summary:
segfault in "${PORTAGE_BIN_PATH}"/filter-bash-environment.py (merging kde-bas...
        









Status:
    

RESOLVED
          WORKSFORME
      






Alias:


        None
    





Product:

Gentoo Linux




Classification:

Unclassified




Component:

[OLD] Core system

  (show other bugs)



Hardware:

All
        Linux
      







Importance:
      
High
       normal
      


Assignee:

Gentoo Linux bug wranglers








URL:







Whiteboard:




Keywords:








Depends on:







Blocks:









 





      Reported:
    
2009-11-23 11:33 UTC by Andreas K. Hüttel





      Modified:
    
2009-11-23 12:30 UTC
      (History)
    




          CC List:
        

0 
          users
        








See Also:






Package list:







Runtime testing required:

---























      Attachments
    




output of emerge --info

              (info.txt,4.31 KB,
                text/plain)

            
2009-11-23 11:34 UTC,

            Andreas K. Hüttel




Details





View All

Add an attachment
        (proposed patch, testcase, etc.)
    








Note
              You need to
              log in
              before you can comment on or make changes to this bug.
            

















Description


Andreas K. Hüttel







          2009-11-23 11:33:45 UTC
        

>>> Installing (4 of 5) kde-base/plasma-workspace-4.3.3-r1
 * Cleaning orphaned Python bytecode from /usr/lib64/python2.6/site-packages/PyKDE4 ..
 * Cleaning orphaned Python bytecode from /usr/share/apps/plasma_scriptengine_python ..
 * Compilation and optimization of Python modules for Python 2.6 ...                                                                    [ ok ]
/usr/lib64/portage/bin/ebuild.sh: line 1622:  5673 Segmentation fault      "${PORTAGE_BIN_PATH}"/filter-bash-environment.py "${filtered_vars}"
 *
 * ERROR: kde-base/plasma-workspace-4.3.3-r1 failed.
 * Call stack:
 *       misc-functions.sh, line   18:  Called source '/usr/lib64/portage/bin/ebuild.sh'
 *               ebuild.sh, line 1776:  Called die
 * The specific snippet of code:
 *      preprocess_ebuild_env || \
 *              die "error processing environment"
 *  The die message:
 *   error processing environment
 *
 * If you need support, post the topmost build error, and the call stack if relevant.
 * A complete build log is located at '/var/tmp/portage/kde-base/plasma-workspace-4.3.3-r1/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/kde-base/plasma-workspace-4.3.3-r1/temp/environment'.
 *
!!! post postinst failed; exiting.
!!! FAILED postinst: 1


No further useful info in the build log. ?????




Comment 1


Andreas K. Hüttel







          2009-11-23 11:34:53 UTC
        

Created attachment 210964 [details]
output of emerge --info




Comment 2


Dawid Węgliński (RETIRED)






          2009-11-23 11:49:33 UTC
        

Can try installing it again, to see if the segfault occurs again in the
same place and report it in this bug?




Comment 3


Andreas K. Hüttel







          2009-11-23 11:57:04 UTC
        

I was re-emerging the package already. No, the problem is not reproducible. The second time everything worked fine.

Some sort of Heisenbug. Let's leave it NEEDINFO for now.
I remember seeing this message before once already, but have no idea anymore what package that was...





Comment 4


Dawid Węgliński (RETIRED)






          2009-11-23 12:30:10 UTC
        

Exactly. As i've already seen such errors before they were 100% nonreproducible. It looks more like random hardware failure. I'd run memtest to see if it may be RAM problem.




Comment 5


Dawid Węgliński (RETIRED)






          2009-11-23 12:30:28 UTC
        

Closing as WORKSFORME.









Format For Printing
 - XML
 - Clone This Bug
 - Clone In The Same 
                        Product
 - Top of page 







Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]

